scripts/integrity_check.py

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/env python3
+#
+# This file is part of Mbed TLS (https://tls.mbed.org)
+#
+# Copyright (c) 2018, Arm Limited, All Rights Reserved
+#
+# Purpose
+#
+# This script requires python 3. This script checks for undesired changes
```

Comment:
The Python 3 requirement is already indicated by the first line and isn't the most important thing. Please move that sentence to a further paragraph in the description. Also the description should be a file docstring (in `"""`).